src/Azure.Functions.Cli/Helpers/AzureHelper.cs

```diff
@@ -250,6 +250,26 @@ private static async Task ArmHttpAsync(HttpMethod method, Uri uri, string access
             }
         }
 
+        public static async Task<HttpResult<Dictionary<string, string>, string>> UpdateFunctionAppAuthSettings(Site site, string accessToken)
+        {
+            var url = new Uri($"{ArmUriTemplates.ArmUrl}{site.SiteId}/config/authsettings?api-version={_storageApiVersion}");
```

Comment:
GetStorageAccount, an existing method/ARM call, uses it. Authsettings requires a later API version than appsettings, because I got the following error with the older version: "The current api-version doesn't support RBAC users.  Please use a newer version instead"

I could either change the name of the variable to something more generic, or add a second variable like _authSettingsApi and hardcode it to 2018-02-01, which is what the storage api version is.